wpilibNewCommands/src/main/native/include/frc2/command/button/Trigger.h

```diff
@@ -57,6 +57,25 @@ class Trigger {
 
   Trigger(const Trigger& other);
 
+  /**
+   * Starts the given command whenever the trigger just becomes active.
+   * 
+   * <p>Takes a raw pointer, and so is non-owning; users are responsible for the
+   * lifespan of the command.
+   * 
+   * @param command The command to bind.
+   * @return The trigger, for chained calls.
+  */
+  Trigger OnChange(Command* command);
+
+  /**
+   * Starts the given command whenever the trigger just becomes active.
+   * 
+   * @param command The command to bind.
+   * @return The trigger, for chained calls.
+  */
+  Trigger OnChange(CommandPtr&& command);
```

Comment:
Why use an rvalue reference here?

Reply:
I'm sorry, I am not very experienced with .cpp or .h, I was just using the OnTrue and OnFalse for reference. What should I do instead?

Reply:
If this is how OnTrue and OnFalse do it then I would assume its correct, I was mostly just curious as to why

Reply:
I am as well, but the pattern is consistent in onTrue and onFalse, so it must be fine to do.